### PR TITLE
AEA-1822: Add more query options to EPSAT.

### PR DESCRIPTION
**Pull request by mfjarvis**

### CODE REVIEW

File: tool/site/client-react/src/components/prescription-tracker/detail/prescriptionDetails.tsx [hunk 1]
```diff
@@ -34,34 +33,41 @@ export function createPrescriptionDetailProps(task: Task): PrescriptionDetailPro
...
-        <SummaryList.Value>{prescription.type}</SummaryList.Value>
+        <SummaryList.Value>{type}</SummaryList.Value>
       </SummaryList.Row>
       <SummaryList.Row>
         <SummaryList.Key>NHS Number</SummaryList.Key>
-        <SummaryList.Value>{prescription.patientNhsNumber}</SummaryList.Value>
+        <SummaryList.Value>{patientNhsNumber}</SummaryList.Value>
+      </SummaryList.Row>
+      {pharmacy &&
+      <SummaryList.Row>
```
Comment (dan-jones-iw) [inline tool/site/client-react/src/components/prescription-tracker/detail/prescriptionDetails.tsx:59]: not that its a big deal, but i missed the `pharmacy &&` initially cause the indentation isnt preserved

File: tool/site/client-react/src/pages/prescriptionSearchPage.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        mimeType="application/json"
+      />
+      <ButtonList>
+        <Button secondary onClick={handleReset}>Back</Button>
+      </ButtonList>
+    </>
+  }
+
+  //TODO - move to separate component
+  return (
```
[the remark this answers is not in the document]
Comment (mfjarvis) [reply tool/site/client-react/src/pages/prescriptionSearchPage.tsx:146]: Yeah the ordering is priority based atm. Will look into it if we make a common component for pages

File: tool/site/client-react/src/pages/prescriptionSearchPage.tsx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+  patientId?: string
+}
+
+const PrescriptionSearchPage: React.FC<PrescriptionSearchPageProps> = ({
+  baseUrl,
+  prescriptionId
+}) => {
+  const [searchResults, setSearchResults] = useState<Bundle>()
+  const [selectedPrescription, setSelectedPrescription] = useState<Task>()
+  const [errorMessage, setErrorMessage] = useState<string>()
```
[the remark this answers is not in the document]
Comment (mfjarvis) [reply tool/site/client-react/src/pages/prescriptionSearchPage.tsx:35]: Yeah getting serious deja vu

File: tool/site/client-react/src/components/prescription-tracker/detail/prescriptionDetails.tsx (hunk 1 above)
Comment (mfjarvis) [reply tool/site/client-react/src/components/prescription-tracker/detail/prescriptionDetails.tsx:59]: Fixed

File: tool/site/client-react/src/pages/prescriptionSearchPage.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (mfjarvis) [reply tool/site/client-react/src/pages/prescriptionSearchPage.tsx:35]: Will give it a go, but separate PR I think

[the remark this answers is not in the document]
Comment (mfjarvis) [reply tool/site/client-react/src/pages/prescriptionSearchPage.tsx:36]: Done in #642

File: tool/site/client-react/src/pages/prescriptionSearchPage.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (mfjarvis) [reply tool/site/client-react/src/pages/prescriptionSearchPage.tsx:149]: Done in #642

File: tool/site/client-react/src/pages/prescriptionSearchPage.tsx [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+    setLoadingMessage(undefined)
+  }
+
+  const initialValues = {
+    prescriptionId: prescriptionId ?? "",
+    patientId: ""
+  }
+
+  if (errorMessage) {
+    return <>
```
[the remark this answers is not in the document]
Comment (mfjarvis) [reply tool/site/client-react/src/pages/prescriptionSearchPage.tsx:95]: Done in #642

**Review by henryisb — APPROVED** [review]

**Pull request merged**